### DISCUSSION TITLE
Implementing real-time object detection on MuSHR

### DISCUSSION THREAD

**Discussion by yukijx** [prl-mushr/mushr#107] [Q&A]
Hello! 

I've been working on the MuSHR software and am wanting to implement real-time object detection using the realsense depth camera installed on the MuSHR car. I’m curious how MuSHR implements its object detection because it runs on Python2.7 while YOLOv5 requires at least the Python 3.7.0 version. 

I’m sure there is something I am missing, and I would really appreciate any direction and advice anyone may have regarding running YOLO onboard the car.

Thank you in advance!

**Comment by maximericaud** [comment]
Hi!
I am also currently working on real-time object detection on MuSHR.
I managed to get YoLov5 to work with python3.6, however I started from scratch by downloading JetPack4.6.1 but not sure if this is required.
[MuSHR installation for YOLOv5.txt](https://github.com/prl-mushr/mushr/files/11925405/MuSHR.installation.for.YOLOv5.txt)
